ingenialink/virtual/dictionary.py

```diff
@@ -0,0 +1,65 @@
+import xml.etree.ElementTree as ET
+from typing import Optional
+
+import ingenialogger
+
+from ingenialink.ethernet.dictionary import EthernetDictionary
+from ingenialink.ethernet.register import EthernetRegister
+
+logger = ingenialogger.get_logger(__name__)
+
+
+class VirtualDictionary(EthernetDictionary):
+    """Contains all registers and information of a dictionary compatible with the virtual drive.
+
+    It adapts a canopen dictionary to work in the ethernet communication used for the virtual drive.
+
+    Args:
+        dictionary_path: Path to the Ingenia dictionary.
+
+    """
+
+    def _read_xdf_register(self, register: ET.Element) -> Optional[EthernetRegister]:
+        current_read_register = super()._read_xdf_register(register)
+
+        if current_read_register is None:
+            return None
+
+        if self.interface == "CAN" and (
+            register.attrib["cat_id"] == "CIA402" or register.attrib["id"].startswith("CIA402_")
+        ):
+            return None
+
+        try:
+            if self.interface == "CAN":
+                reg_address = int(register.attrib["address"][:6], 16)
```

Comment:
We could use [this ](https://github.com/ingeniamc/ingenialink-python/blob/master/ingenialink/canopen/servo.py#L142) function